instrumented/benchmark/build.gradle.kts

```diff
@@ -100,12 +100,55 @@ kotlinConfig()
 detektConfig()
 ktLintConfig()
 
+// Logs Benchmarks
 reviewBenchmark {
-    addThreshold("benchmark_writing_logs", TimeUnit.MICROSECONDS.toNanos(500))
-    addThreshold("benchmark_writing_logs_with_attributes", TimeUnit.MILLISECONDS.toNanos(1))
-    addThreshold("benchmark_writing_logs_with_tags", TimeUnit.MILLISECONDS.toNanos(1))
-    addThreshold("benchmark_writing_logs_with_throwable", TimeUnit.MILLISECONDS.toNanos(1))
+    addThreshold(
+        "benchmark_writing_logs",
+        TimeUnit.MICROSECONDS.toNanos(500)
+    )
+    addThreshold(
+        "benchmark_writing_logs_with_attributes",
+        TimeUnit.MILLISECONDS.toNanos(1)
+    )
+    addThreshold(
+        "benchmark_writing_logs_with_tags",
+        TimeUnit.MILLISECONDS.toNanos(1)
+    )
+    addThreshold(
+        "benchmark_writing_logs_with_throwable",
+        TimeUnit.MILLISECONDS.toNanos(1)
+    )
+
+    addThreshold(
+        "benchmark_sending_medium_load_of_logs",
+        TimeUnit.MILLISECONDS.toNanos(200)
+    )
+    addThreshold(
+        "benchmark_sending_heavy_load_of_logs",
+        TimeUnit.MILLISECONDS.toNanos(300)
+    )
+}
 
```

Comment:
Why do you create a second `reviewBenchmark` block ? That's a bit unusual, why not have a single block and just add comment to distinguish logs from traces ?

Reply:
was hesitating, I thought is much more visible this way....will put it back, you're right ;)